paying_for_college/tests/test_views.py

```diff
@@ -116,12 +116,13 @@ def test_feedback(self):
         self.assertTrue(sorted(response.context_data.keys()) ==
                         ['base_template', 'form', 'url_root'])
 
-    @mock.patch('paying_for_college.views.render_to_response')
-    def test_feedback_post(self, mock_render):
-        client.post(
+    @mock.patch('paying_for_college.views.Feedback')
+    def test_feedback_post(self, mock_feedback):
+        response = client.post(
             reverse('disclosures:pfc-feedback'),
             data=self.feedback_post_data)
-        self.assertTrue(mock_render.call_count == 1)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue(mock_feedback.call_count == 1)
```

Comment:
This seems fine. One minor suggestion: this test seems to be testing that the `Feedback` constructor is called when this view is `POST`ed to. This seems a bit specific, for example the view might create the feedback object in some other way (e.g. `Feedback.objects.create(...)`). It might be more clarifying to rename this test slightly more specifically (like `test_feedback_post_creates_feedback`) and then do something like `self.assertFalse(Feedback.objects.exists())` before and then `self.assertTrue(Feedback.objects.exists())` after.